frontend/src/tests/components/Leaderboard/LeaderboardTable.test.js

```diff
@@ -96,19 +92,25 @@ describe("LeaderboardTable tests", () => {
 
   });
 
-  test("Total wealth is formatted correctly", () => {
+  test("Table is formatted correctly", () => {
     const currentUser = currentUserFixtures.adminUser;
 
     render(
       <QueryClientProvider client={queryClient}>
         <MemoryRouter>
-          <LeaderboardTable leaderboardUsers={leaderboardFixtures.threeUserCommonsLB} currentUser={currentUser} />
+          <LeaderboardTable leaderboardUsers={leaderboardFixtures.fiveUserCommonsLB} currentUser={currentUser} />
```

Comment:
If you remove a prop from a component, also remove any existing components that pass the prop to it. There's a couple more.
```suggestion
          <LeaderboardTable leaderboardUsers={leaderboardFixtures.fiveUserCommonsLB} />
```

Reply:
Agreed with Chris..  You can commit this suggestion by just clicking the `Commit suggestion` box.  

<img width="183" alt="image" src="https://github.com/ucsb-cs156-f23/proj-happycows-f23-5pm-4/assets/1119017/d669632d-5ef9-4737-b79f-d037c813c38e">